### PR TITLE
#4: Translate the main rules into English and post them in CONTRIBUTE.md file

### PR DESCRIPTION
**Pull request by katridi**
Some basic conventions about teamwork and decision making

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+2. We work as a team. Do not compete and not measure by any quality, except how well we give support to others and how we share our knowledge. Help your colleagues is the most important KPI!
+
+3. We speak what we think and do accordingly. [Self-] deception, especially in measuring own abilities does not make any sense: everything here exists to learn something new, but one needs to know basics.
+
+4. Nobody owes anything, except what he had promised. Consequently, we do not promise what we can not handle or not going to do.
+
+5. Point 4 is very important. We understand that everyone has his main work, study, family, and other bunch sources of unforeseen circumstances. If something affects work to be done we do not argue but try to find ways to help person particularly and overall project. If such "unforeseen" occur too often - we discuss how much time this person can truly spend on the project and act accordingly.
```
Comment (asechnaya) [inline CONTRIBUTING.md]: If such "unforeseen" occurs too often

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+6. Code quality is the most important characteristic. It is better longer than the worst. It is worth to ask for help from your teammate than insist on code that you are not quite sure. We respect each other and do not declare code finished, which does not meet basic quality requirements (does not compile, launch, cover adequately by tests, fit with basic style guides, etc).
+
+7. All codebase is our common code. Everyone threat his code as critical as someone's else code does not protect it from legitimate criticism. Eager for someone to criticize his code and thank for such criticizing.
+
+8. We accept https://github.com/mvartanyan and https://github.com/jekoff as an imaginary customer voice. Technical decision making we make together.
```
Comment (asechnaya) [inline CONTRIBUTING.md]: We make technical decisions together.

File: CONTRIBUTING.md (hunk 1 above)
Comment (asechnaya) [reply CONTRIBUTING.md]: use a comma (not a dash)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Contributing to lektorium
```
Comment (mjecov) [inline CONTRIBUTING.md]:
```suggestion
# Contributing to Lektorium
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (mjecov) [inline CONTRIBUTING.md]:
```suggestion
8. We accept [Michael](https://github.com/mvartanyan) and [Max](https://github.com/jekoff) as an imaginary customer voice. Technical decision making we make together.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+
+## What should I know before I get started?
+1. We treat everyone and each as a friend and with respect. Particularly: taking it personally, insulting, indignity by any reason, critics and compilation personal attributes, which receiver cannot change (such as sex, age, health condition, sexual orientation, etc) are not allowed. We keep close to each other, joke, fool around and have fun from interaction, but in the frame what's written above.
```
Comment (asechnaya) [inline CONTRIBUTING.md]: in term (or terms) written above (no "what's", maybe that. but that can be omitted)

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+2. We work as a team. Do not compete and not measure by any quality, except how well we give support to others and how we share our knowledge. Help your colleagues is the most important KPI!
+
+3. We speak what we think and do accordingly. [Self-] deception, especially in measuring own abilities does not make any sense: everything here exists to learn something new, but one needs to know basics.
+
+4. Nobody owes anything, except what he had promised. Consequently, we do not promise what we can not handle or not going to do.
```
Comment (asechnaya) [inline CONTRIBUTING.md]: we do not promise => we don't make promises

File: CONTRIBUTING.md (hunk 1 above)
Comment (asechnaya) [inline CONTRIBUTING.md]: If I were you, I would say "main job" instead of "main work".

Comment (asechnaya) [reply CONTRIBUTING.md]: Everyone has his *own*

File: CONTRIBUTING.md (hunk 4 above)
Comment (mvartanyan) [inline CONTRIBUTING.md]:
```suggestion
1. We treat everyone and each as a friend and with respect. In particular: personal attacks, insulting, indignity for any reason, critics and stereotyping on the basis of personal attributes the receiver cannot change (such as gender, age, health condition, sexual orientation, etc) are not allowed. We keep close to each other, joke, fool around and have fun from our interaction, keeping within the limits described above.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+
+## What should I know before I get started?
+1. We treat everyone and each as a friend and with respect. Particularly: taking it personally, insulting, indignity by any reason, critics and compilation personal attributes, which receiver cannot change (such as sex, age, health condition, sexual orientation, etc) are not allowed. We keep close to each other, joke, fool around and have fun from interaction, but in the frame what's written above.
+
+2. We work as a team. Do not compete and not measure by any quality, except how well we give support to others and how we share our knowledge. Help your colleagues is the most important KPI!
```
Comment (mvartanyan) [inline CONTRIBUTING.md]:
```suggestion
2. We work as a team. Do not compete or brag about any of our qualities, except the one of how well we support others and how we share our knowledge. Help to your colleagues is the most important KPI!
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+3. We speak what we think and do accordingly. [Self-] deception, especially in measuring own abilities does not make any sense: everything here exists to learn something new, but one needs to know basics.
```
Comment (mvartanyan) [inline CONTRIBUTING.md]:
```suggestion
3. We say what we think and do what we say. [Self-] deception, especially in assessing own abilities does not make any sense: everything is here to learn something new, and in order to improve one needs to know the true baseline.
```

File: CONTRIBUTING.md (hunk 5 above)
Comment (mvartanyan) [inline CONTRIBUTING.md]:
```suggestion
4. We don't owe anything to one anothe, except what we had promised. Consequently, we do not promise what we can not handle or are not going to do.
```

this is a part of the previous point

File: CONTRIBUTING.md (hunk 2 above)
Comment (katridi) [reply CONTRIBUTING.md]: Yes, thanks! Forgot to make hyperlinks

File: CONTRIBUTING.md (hunk 1 above)
Comment (mvartanyan) [inline CONTRIBUTING.md]:
```suggestion
5. While point 4 above is very important, we understand that everyone has their day job, study, family, and other  sources of unforeseen circumstances. If something affects work to be done, we do not quarrel but try to find ways to help our colleague and the project. If such "oopses" occur too often, we review how much time this colleague can actually contribute to the project, and act accordingly.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+4. Nobody owes anything, except what he had promised. Consequently, we do not promise what we can not handle or not going to do.
+
+5. Point 4 is very important. We understand that everyone has his main work, study, family, and other bunch sources of unforeseen circumstances. If something affects work to be done we do not argue but try to find ways to help person particularly and overall project. If such "unforeseen" occur too often - we discuss how much time this person can truly spend on the project and act accordingly.
+
+6. Code quality is the most important characteristic. It is better longer than the worst. It is worth to ask for help from your teammate than insist on code that you are not quite sure. We respect each other and do not declare code finished, which does not meet basic quality requirements (does not compile, launch, cover adequately by tests, fit with basic style guides, etc).
```
Comment (mvartanyan) [inline CONTRIBUTING.md]:
```suggestion
6. Code quality is its most important characteristic. It is always better to take longer than to contribute produce substandard code. It is worth to ask for help from your teammate than declare code you are not sure about as done. We respect each other's time and we do not declare code finished unless it meets basic quality requirements (it builds, runs, is adequately covered with tests, complies to basic style guides, etc).
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+7. All codebase is our common code. Everyone threat his code as critical as someone's else code does not protect it from legitimate criticism. Eager for someone to criticize his code and thank for such criticizing.
```
Comment (mvartanyan) [inline CONTRIBUTING.md]:
```suggestion
7. The whole codebase belongs to all of us. Everyone looks at their own code as critically as at someone else's, and  does not defend own code from legitimate criticism. The author is always eager to receive critique of their code and is grateful for such critique.
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+8. We accept https://github.com/mvartanyan and https://github.com/jekoff as an imaginary customer voice. Technical decision making we make together.
+
+9. All proposals may and must be expressed, considered and argued. When the decision is established we are moving accordingly without deviation, except something previously was unknown what affects on the entire landscape. In such cases we reconsider the decision. It happens and that is totally fine!
```
Comment (mvartanyan) [inline CONTRIBUTING.md]:
```suggestion
9. Any proposals can and should be voiced, considered and debated. When the decision is taken,  we are moving accordingly without deviation, except something previously unknown affects on the context in which the decision was taken. In such cases, we reconsider the decision. It happens and that is totally fine!
```

**Review by mvartanyan — CHANGES_REQUESTED** [review]
added some adjustments, mostly editing.

File: CONTRIBUTING.md (hunk 5 above)
Comment (asechnaya) [reply CONTRIBUTING.md]: _measure **up in**_

Comment (asechnaya) [reply CONTRIBUTING.md]: _we give support to others_  maybe _we support others_

Comment (asechnaya) [reply CONTRIBUTING.md]: We don't owe anything to one another

Comment (katridi) [reply CONTRIBUTING.md]: Done

**Review by mvartanyan — APPROVED** [review]

**Review by mjecov — APPROVED** [review]

**Pull request merged**